www/apps/docs/content/upgrade-guides/medusa-ui/2-0-0.md

```diff
@@ -0,0 +1,105 @@
+---
+description: "Actions Required for v.2.0.0"
+sidebar_custom_props:
+  iconName: "component-solid"
+---
+
+# v2.0.0
+
+Version 2.0.0 of Medusa UI introduces a set of new components, and breaking changes to some of the existing components.
+
+## How to Update
+
+Run the following command in your project:
+
+```bash npm2yarn
+npm install @medusajs/ui@2.0.0
+```
+
+---
+
+## Breaking Changes
+
+### `Button` Component
+
+The [`Button`](https://docs.medusajs.com/ui/components/button) component has been split into two components: `Button` and `IconButton`.
```

Comment:
Due to the copy action on inline code, can you remove the inline code formatting for all links that have code in them?

```suggestion
The [Button](https://docs.medusajs.com/ui/components/button) component has been split into two components: `Button` and `IconButton`.
```